comps/retrievers/src/opea_retrievers_microservice.py

```diff
@@ -59,7 +61,7 @@
 )
 @register_statistics(names=["opea_service@retrievers"])
 async def retrieve_docs(
-    input: Union[EmbedDoc, EmbedMultimodalDoc, RetrievalRequest, ChatCompletionRequest],
+    input: Union[EmbedDoc, EmbedMultimodalDoc, RetrievalRequest,  RetrievalRequestArangoDB, ChatCompletionRequest],
```

Comment:
question: is this all that is required in order for the Retriever service to accept inputs of type `RetrievalRequestArangoDB`?